src/ai/backend/storage/dellemc/onefs_client.py

```diff
@@ -37,19 +36,14 @@ def __init__(
         password: str,
         *,
         api_version: str = "12",
-        system_name: str = "nfs",
+        system_name: Optional[str] = "nfs",
```

Comment:
What is the role of this parameter? How/where do we derive it from cluster configuration?

Reply:
I checked that the parameter is used when this client queries workload stats of OneFS. I changed this parameter as nullable since the workload stat API does not require `system_name` parameter. I left the default value as the same to reduce side effects.